docs/reference/ml/anomaly-detection/ml-configuring-aggregations.asciidoc

```diff
@@ -45,6 +45,12 @@ finer, more granular time buckets, which are ideal for this type of analysis. If
 your detectors use <<ml-count-functions,count>> or <<ml-rare-functions,rare>>
 functions, set the interval to the same value as the bucket span.
 
+If your {dfeed} uses aggregations and model plot is not enabled for the 
```

Comment:
This limitation also applies to the anomaly charts in the Anomaly Explorer, with or without model plot enabled for 7.10.